exo-run/src/service-restarter.ls

```diff
@@ -30,19 +30,21 @@ class ServiceRestarter extends EventEmitter
 
   _restart: ->
     @watcher.close!
-    cwd = @docker-config-location 
-    DockerCompose.kill-container {service-name: @role, cwd, @write}, (exit-code) ~>
-      | exit-code => @emit 'error', "Docker failed to kill container #{@role}"
-      @write "Docker container stopped"
-
-      DockerCompose.create-new-container {service-name: @role, cwd, @env, @write}, (exit-code) ~>
-        | exit-code => @emit 'error', "Docker image failed to rebuild #{@role}"
-        @write "Docker image rebuilt"
-
-        DockerCompose.start-container {service-name: @role, cwd, @env, @write}, (exit-code) ~>
-          | exit-code => @emit 'error', "Docker container failed to restart #{@role}"
-          @watch!
-          @logger.log {role: 'exo-run', text: "'#{@role}' restarted successfully"}
+    set-timeout ( ~>
+      cwd = @docker-config-location 
+      DockerCompose.kill-container {service-name: @role, cwd, @write}, (exit-code) ~>
+        | exit-code => @emit 'error', "Docker failed to kill container #{@role}"
+        @write "Docker container stopped"
+
+        DockerCompose.create-new-container {service-name: @role, cwd, @env, @write}, (exit-code) ~>
+          | exit-code => @emit 'error', "Docker image failed to rebuild #{@role}"
+          @write "Docker image rebuilt"
+
+          DockerCompose.start-container {service-name: @role, cwd, @env, @write}, (exit-code) ~>
+            | exit-code => @emit 'error', "Docker container failed to restart #{@role}"
+            @watch!
+            @logger.log {role: 'exo-run', text: "'#{@role}' restarted successfully"})
+    , 2500
```

Comment:
2500, meaning 2.5 seconds? From experience working on an exosphere project, I'm worried this is too long. Then again, it's hard to guess what the right amount of time would be. Maybe this is fine for now and we can tune it later? 

In either case, it seems like you're intentionally using the same number here and for the `stabilityThreshold` above. Maybe we should set this to a class variable in the constructor, like `@timeDelay = 2500` or something?